INSTALL.md

```diff
@@ -368,7 +368,16 @@ cd filter_audio
 make -j$(nproc)
 sudo make install
 ```
-
+### Sqlcipher compiling, For Fedora users and possibly others.
+```bash
+git clone https://github.com/sqlcipher/sqlcipher
+cd sqlcipher
+autoreconf -if
+./configure
+make -j$(nproc)
+sudo make install
+cd ..
+````
```

Comment:
@zetok, more like:

### sqlcipher (Fedora)  ?

Also, is the empty line doing anything programmatically  otherwise it  looks like crap with it there.